### DISCUSSION TITLE
Problem with delete commands

### DISCUSSION THREAD

**Discussion by matt1600** [jchristn/Less3#13] [Q&A]
Could definitely be me doing something wrong, but I'm having trouble with deletion. Creating buckets and uploading objects to them has been working. 

<img width="960" alt="Screenshot 2024-06-24 at 9 49 49 AM" src="https://github.com/jchristn/Less3/assets/17398480/1470e2e1-b548-4ebe-ab40-9707c0c6e982">
<img width="766" alt="Screenshot 2024-06-24 at 9 45 44 AM" src="https://github.com/jchristn/Less3/assets/17398480/f0f733fc-670c-4f94-a70f-2d0d451a51a5">

**Comment by jchristn** [comment]
Sorry, I never got a notification on this.  Are you using path style or virtual hosted request style?
